src/net/sourceforge/kolmafia/request/UseItemRequest.java

```diff
@@ -6213,6 +6213,14 @@ else if (responseText.contains("You can't figure out the angle")) {
               });
           break;
         }
+      case ItemPool.MINIATURE_EMBERING_HULK:
+
+        // If we are redirected to a fight, the item is
+        // consumed elsewhere. If we got here, it wasn't
+        // actually consumed
+
+        Preferences.setBoolean("_emberingHulkFought", true);
+        return;
```

Comment:
I don't understand the comment or am missing something in context.  Simplistically "if we got here it wasn't actually consumed" suggests to me that if we got here it wasn't consumed so the preference needs to be set to or remain at false.  That is not what the code does.  I won't make this a blocker because I think the problem is my understanding and not the code.

Reply:
If we use the item when we've already fought a hulk today, the item isn't consumed. If we try to use the item again, it'll fail because we've already fought one (perhaps outside of Mafia).

Reply:
I think I get it.  Thanks.